src/main.js

```diff
@@ -35,6 +35,23 @@ class ameClient {
             throw err;
         }
     }
+    
+
+    async image(endpoint, data = {}){
+      if(!endpoint) throw 'Missing endpoint';
+      
+      try {
+            let image = await get(`${this.baseURL}/image/${endpoint}`, data, {
```

Comment:
The correct format is this for get 😄 
```suggestion
            let image = await get(`${this.baseURL}/image/${endpoint}`, {
                params: {
                    data
                },
            
```